src/main/java/com/avispl/symphony/dal/avdevices/power/surgex/defenderseries/common/constants/Constant.java

```diff
@@ -0,0 +1,50 @@
+package com.avispl.symphony.dal.avdevices.power.surgex.defenderseries.common.constants;
+
+
+/**
+ * Utility class that defines constant values used across the application.
+ *
+ * @author Kevin / Symphony Dev Team
+ * @since 1.0.0
+ */
+public class Constant {
+	private Constant() {
+		// Prevent instantiation
```

Comment:
This class could've been an interface if you want to prevent its instantiation and only use it as constants storage :)

Reply:
It's a good idea :D, but I think the main purpose of the interface is to contain behavior for other implemented classes, not only to contain the constants, so I'll keep this class as is

Reply:
@ThanhD-AVISPL yeah, that's a fair point, but for us it's merely a tool that we can use, so it doesn't really matter which component will we use for the matter - class, interface or enum, for instance. But it's not a strong opinion, just a way to avoid the necessity of declaring a private constructor :)